ui/org.eclipse.pde.launching/src/org/eclipse/pde/internal/launching/JUnitLaunchValidationOperation.java

```diff
@@ -78,6 +123,27 @@ private void addError(String message) {
 		fErrors.put(message.replaceAll("\\R", " "), null); //$NON-NLS-1$//$NON-NLS-2$
 	}
 
+	@SuppressWarnings("restriction")
+	private static Map<String, String> junitBundleNames() {
+		Map<String, String> map = new HashMap<>();
+		map.put("org.junit.jupiter.api", BuildPathSupport.JUNIT_JUPITER_API); //$NON-NLS-1$
+		map.put("org.junit.jupiter.engine", BuildPathSupport.JUNIT_JUPITER_ENGINE); //$NON-NLS-1$
+		map.put("org.junit.platofrm.engine", BuildPathSupport.JUNIT_PLATFORM_ENGINE); //$NON-NLS-1$
+		map.put(BuildPathSupport.JUNIT_JUPITER_API, BuildPathSupport.JUNIT_JUPITER_API);
```

Comment:
Since we want to report the bundle that depends on JUnit 6, we can no longer just look for `junit-jupiter-engine`. Since we find that in the dependencies of `junit-jupiter-api`, which we must skip for the report to make sense - we are not reporting the whole chain of dependencies.

The report then looks like this:

<img width="1084" height="278" alt="Screenshot_20251104_134117" src="https://github.com/user-attachments/assets/46ea5cd8-f2dc-41a1-992e-8d0aaa523c71" />